### PR TITLE
Enable log tracing information for RTI

### PR DESCRIPTION
**Pull request by chanijjani**

### CODE REVIEW

**Comment by chanijjani** [comment]
Need to rebase after https://github.com/hokeun/lf-rust-rti/pull/41 is merged.

**Comment by chanijjani** [comment]
Tracing results with HelloDistributed.lf
[trace_svg.zip](https://github.com/hokeun/lf-rust-rti/files/14423819/trace_svg.zip)

File: rust/rti/src/trace.rs [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+        }
+    }
+}
+
+#[derive(Debug, Clone, Copy)]
+pub struct TraceRecord {
+    event_type: u64,
+    pointer: Void, // pointer identifying the record, e.g. to self struct for a reactor.
+    src_id: i32,   // The ID number of the source (e.g. worker or federate) or -1 for no ID number.
+    dst_id: i32, // The ID number of the destination (e.g. reaction or federate) or -1 for no ID number.
```
Comment (hokeun) [inline rust/rti/src/trace.rs:117]: It's very minor, but the indentation of the comment at this line seems off.

**Review by hokeun — APPROVED** [review]
🚀

File: rust/rti/src/trace.rs (hunk 1 above)
Comment (chanijjani) [reply rust/rti/src/trace.rs:117]: Thank you! I'll apply `cargo fmt` again.

**Pull request merged**